core/src/main/java/io/seata/core/rpc/netty/NettyRemotingServer.java

```diff
@@ -23,43 +23,105 @@
 import io.netty.handler.timeout.IdleState;
 import io.netty.handler.timeout.IdleStateEvent;
 import io.seata.common.util.NetUtil;
+import io.seata.core.protocol.HeartbeatMessage;
+import io.seata.core.protocol.MessageType;
+import io.seata.core.protocol.ProtocolConstants;
 import io.seata.core.protocol.RpcMessage;
-import io.seata.core.rpc.ChannelManager;
+import io.seata.core.rpc.RegisterCheckAuthHandler;
 import io.seata.core.rpc.RemotingServer;
 import io.seata.core.rpc.RpcContext;
 import io.seata.core.rpc.TransactionMessageHandler;
-import io.seata.core.rpc.processor.Pair;
 import io.seata.core.rpc.processor.RemotingProcessor;
+import io.seata.core.rpc.processor.Pair;
+import io.seata.core.rpc.processor.server.RegRmProcessor;
+import io.seata.core.rpc.processor.server.RegTmProcessor;
+import io.seata.core.rpc.processor.server.ServerHeartbeatProcessor;
+import io.seata.core.rpc.processor.server.ServerOnRequestProcessor;
+import io.seata.core.rpc.processor.server.ServerOnResponseProcessor;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.ThreadPoolExecutor;
+import java.util.concurrent.TimeoutException;
 
 /**
- * The type Rpc remoting server.
+ * The netty remoting server.
  *
  * @author slievrly
  * @author xingfudeshi@gmail.com
  * @author zhangchenghui.dev@gmail.com
  */
-public abstract class AbstractRpcRemotingServer extends AbstractRpcRemoting implements RemotingServer {
+public class NettyRemotingServer extends AbstractNettyRemoting implements RemotingServer {
```

Comment:
Keep abstract, there will be netty Server between TC and TC, don't write business logic dead here

Reply:
You are right, I will roll back later.

Reply:
done.